fastlane/lib/fastlane/actions/device_grid/BuildAndUploadAppetize

```diff
@@ -0,0 +1,22 @@
+desc "Generate and upload an .app to appetize.io"
+desc "Import this lane and call it from Danger"
+
+private_lane :build_and_upload_appetize do |options|
+  tmp_path = "/tmp/fl_build"
+
+  xcodebuild_configs = options[:xcodebuild]
+  xcodebuild_configs[:sdk] = "iphonesimulator"
+  xcodebuild_configs[:derivedDataPath] = tmp_path
+  
+  xcodebuild(xcodebuild_configs)
+
+  app_path = Dir[File.join(tmp_path, "**", "*.app")].last
+  UI.user_error("Couldn't find app") unless app_path
+
+  zipped_ipa = zip(path: app_path, output_path: File.join(tmp_path, "Result.zip"))
+
+  appetize(path: zipped_ipa)
+
+  File.write("public_key.txt", lane_context[SharedValues::APPETIZE_PUBLIC_KEY])
```

Comment:
this file name is very generic. `appetize_public_key.txt` ? (same in the load part)